src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -815,8 +826,21 @@ def run(self):
         self.cli_ctx.get_progress_controller().init_progress(ShellProgressView())
         self.cli_ctx.get_progress_controller = self.progress_patch
 
+        # load command table
         self.command_table_thread = LoadCommandTableThread(self.restart_completer, self)
         self.command_table_thread.start()
+        self.command_table_thread.start_time = time.time()
+        print_styled_text([(Style.ACTION, "Loading command table... Expected time is around 1 minute.")])
+        time_spent_on_loading = 0
+        # still loading commands, show the time of loading
+        while self.command_table_thread.is_alive() and time_spent_on_loading < 150:
+            time_spent_on_loading = time.time() - self.command_table_thread.start_time
+            # print out loading time every 10 seconds
+            if time_spent_on_loading % 10 < 1:
+                print_styled_text([(Style.ACTION, "Already spent {} seconds on loading command table.".format(round(time_spent_on_loading, 1)))])
+            time.sleep(1)
```

Comment:
Why not using `time.sleep(10)`
```suggestion
        while self.command_table_thread.is_alive() and time_spent_on_loading < 150:
            time_spent_on_loading = time.time() - self.command_table_thread.start_time
            print_styled_text([(Style.ACTION, "Already spent {} seconds on loading command table.".format(round(time_spent_on_loading, 1)))])
            time.sleep(10)
```

Reply:
Already modified in https://github.com/wu5bocheng/azure-cli-extensions/pull/3
Please review the branch first.

Because I didn't setup new branches for different functions initially. The first few PRS might be confusing.